src/location-search/LocationSearch.tsx

```diff
@@ -27,6 +27,9 @@ import {LocationSearchNavigationProp} from './';
 import LocationResults from './LocationResults';
 import {LocationSearchResult} from './types';
 import useDebounce from './useDebounce';
+import {useTranslation} from '../utils/language';
+import {LocationSearchTexts} from '../translations';
+import {TranslateFunction} from '../translations/utils';
```

Comment:
To give more context to the discussion from previous comments: I think this is semantically the same module and should be retrievable from the same file as "public API".

Reply:
:+1: